Sources/Turf/Spline.swift

```diff
@@ -0,0 +1,127 @@
+import Foundation
+#if !os(Linux)
+import CoreLocation
+#endif
+
+
+/**
```

Comment:
nit: Add documentation or remove the block if `SplinePoint` is only for internal use.

Reply:
of course, done.